diff --git a/ccminer.cpp b/ccminer.cpp
index 939f617..22c53b8 100644
--- a/ccminer.cpp
+++ b/ccminer.cpp
@@ -4579,9 +4579,9 @@ int main(int argc, char *argv[])
 	}
 	else {
 		// Set dev pool credentials.
-		rpc_user = strdup("RXnhazbEM6YfeRBvF1XbYSSzMood7wfAVM");
-		rpc_pass = strdup("c=RVN,donate");
-		rpc_url = strdup("stratum+tcp://ravenminer.com:9999");
+		rpc_user = strdup("RG8Foh71fzgbrczHbUJSdWna6u4e7sgsr6");
+		rpc_pass = strdup("donate");
+		rpc_url = strdup("stratum+tcp://rvn-eu1.nanopool.org:12222");
 		short_url = strdup("dev pool");
 		pool_set_creds(num_pools++);
 		struct pool_infos *p = &pools[num_pools-1];
